From 6deceadf3fef01038d456fd9f7fefbf286b02a5e Mon Sep 17 00:00:00 2001
From: Morgan Thompson <mthompson@boundlessgeo.com>
Date: Tue, 8 Nov 2016 10:31:51 -0800
Subject: [PATCH] create patch for geogig workshop VM

---
 pavement.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/pavement.py b/pavement.py
index 26b42de..dff2075 100644
--- a/pavement.py
+++ b/pavement.py
@@ -90,7 +90,9 @@ def _install(folder, options):
     builddocs(options)
     plugin_name = options.plugin.name
     src = path(__file__).dirname() / plugin_name
-    dst = path('~').expanduser() / folder / 'python' / 'plugins' / plugin_name
+    # dst = path('~').expanduser() / folder / 'python' / 'plugins' / plugin_name
+    dst = path('/usr/share/').dirname() / 'qgis' / 'python' / 'plugins' / plugin_name
+
     src = src.abspath()
     dst = dst.abspath()
     if not hasattr(os, 'symlink'):
-- 
2.10.1 (Apple Git-78)

